### PR TITLE
(DOCSP-25302) Tutorial for creating ephemeral project and cluster

### PR DESCRIPTION
**Pull request by sarahsimpers**
I tested this process end to end to confirm that it works.

[Jira Ticket](https://jira.mongodb.org/browse/DOCSP-25302)
[Staging: Tutorial](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-25302/atlas-cli-ephemeral-cluster/#std-label-atlas-cli-ephemeral-cluster)
[Staging: Top-level Tutorials Page](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-25302/atlas-cli-tutorials/)
[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64947f91423952aeb96facb4)

### CODE REVIEW

File: source/atlas-cli-ephemeral-cluster.txt
Comment (jwilliams-mongo) [inline source/atlas-cli-ephemeral-cluster.txt:16]: I was just going to ask if we could use a simpler word than ephemeral, but explaining it here is a nice touch

File: source/atlas-cli-ephemeral-cluster.txt
Comment (jwilliams-mongo) [inline source/atlas-cli-ephemeral-cluster.txt:18]: avoid passive voice
```suggestion
you isolate from your production projects and {+clusters+}. You can use
```

File: source/atlas-cli-ephemeral-cluster.txt
Comment (jwilliams-mongo) [inline source/atlas-cli-ephemeral-cluster.txt:29]:
```suggestion
- Create an |service| user account.
```

File: source/atlas-cli-ephemeral-cluster.txt
Comment (jwilliams-mongo) [inline source/atlas-cli-ephemeral-cluster.txt:45]: question: is this returned in a user ID field? Might be worth showing example output

File: source/atlas-cli-ephemeral-cluster.txt
Comment (jwilliams-mongo) [inline source/atlas-cli-ephemeral-cluster.txt:32]: question: do you think it makes sense for us to add an explicit step below that shows users how to get their org id or do you think it should stay in the prereqs?

File: source/atlas-cli-ephemeral-cluster.txt
Comment (jwilliams-mongo) [inline source/atlas-cli-ephemeral-cluster.txt:63]: I think using some sort of formatting here will help users by calling this info out
```suggestion
         project ID is ``64933bde48add154124e343f``.
```

File: submodules/mongodb-atlas-cli
Comment (jwilliams-mongo) [inline submodules/mongodb-atlas-cli:1]: This shouldn't conflict with the submodule updates in the release notes PR, correct? https://github.com/mongodb/docs-atlas-cli/pull/163

File: source/atlas-cli-ephemeral-cluster.txt
Comment (jwilliams-mongo) [inline source/atlas-cli-ephemeral-cluster.txt:73]: does this create a database user?

File: source/atlas-cli-ephemeral-cluster.txt
Comment (jwilliams-mongo) [inline source/atlas-cli-ephemeral-cluster.txt:109]: `a few` is vague. Can we provide a specific recommendation here? Maybe five minutes?
```suggestion
            a few minutes, then run the ``atlas projects delete``
```

**Review by jwilliams-mongo — COMMENTED** [review]
@sarahsimpers a few minor comments and questions, but looks good overall

File: submodules/mongodb-atlas-cli
Comment (sarahsimpers) [reply submodules/mongodb-atlas-cli:1]: It likely will, I'll need to check out the most recent submodule before I can merge this if yours is merged first.

File: source/atlas-cli-ephemeral-cluster.txt
Comment (sarahsimpers) [reply source/atlas-cli-ephemeral-cluster.txt:73]: Yes

File: submodules/mongodb-atlas-cli
Comment (sarahsimpers) [reply submodules/mongodb-atlas-cli:1]: To clarify: it will conflict if yours merges first, I'll need to update it if that's the case. If mine merges first, you should be fine because the changes here are from rebasing against master, so they should be at the most recent submodule before your PR's.

Comment (jwilliams-mongo) [reply submodules/mongodb-atlas-cli:1]: go ahead and merge yours first. I'll deal with conflicts if they arise on my end.

**Review by fmenezes — APPROVED** [review]

**Pull request merged**
